### ISSUE TITLE
pyvisa-py

### ISSUE THREAD

**Issue by greyltc** [conda-forge/pyvisa-feedstock#4]
Hello,
Could you please add pyvisa-py to conda-forge? https://pypi.python.org/pypi/PyVISA-py/0.2

pyvisa-py is a pure python back end for this project (pyvisa).

**Comment by melund** [comment]
That would be great. But you would first need to add the dependencies of pyvisa-py. PyUSB and PySerial. You can probably do it your self. Take a look at https://github.com/conda-forge/staged-recipes

**Comment by greyltc** [comment]
PyUSB and PySerial are optional deps for pyvisa-py. I don't know how that's handled here, I've never packaged anything for conda-forge before.

I've built the package locally with:
```
conda skeleton pypi pyvisa-py
cd pyvisa-py
conda build --no-test .
```
and the build completes (on a python 3.4 win32 system). Testing fails because the module can't be imported (I guess because a hyphen in the module name is forbidden syntax), so I added `--no-test`. I'll have a look at the staging repo, but I guess I'd have to iron out the testing/import issue before I can make a PR there.
